tests/extract/test_dcatus.py

```diff
@@ -25,6 +25,8 @@ def test_extract_bad_url(get_bad_url, create_client):
 
     if str(main(get_bad_url, S3_client)) == "non-200 status code":
```

Comment:
you can just assert `VAL = VAL` and skip the conditional in all these.

Reply:
totally agree. thanks for the recommendation.

Reply:
resolved [here](https://github.com/GSA/datagov-harvesting-logic/commit/17a5d09e3c16f4c30de6d327c88e8b7fbb0c205e)